playbook-website/app/views/guides/getting_started/figma_setup.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Figma Setup
+icon: pen-ruler
+description: Figma offers a 1:1 match to our open-source platform, offering components with customizable properties.
+---
+
+Playbook seamlessly integrates with Figma, providing a 1:1 match to our open-source platform. The library offers meticulously crafted components with adjustable properties, allowing designers to quickly build out complex designs straight out of the box.
+
+![Buttons](https://github.com/powerhome/playbook/assets/146777111/bc4d0c48-b934-4e5a-9f16-9402489c0844)
```

Comment:
@bianca0303 this image is looking a little blurry, can we get a higher res here?